Gemfile

```diff
@@ -5,3 +5,5 @@ gem 'rake', '>= 10.4.2'
 gem 'awestruct', '0.6.0'
 gem 'asciidoctor', '1.5.8' # AsciiDoc
 gem 'json', '2.1.0'
+gem 'pandoc-ruby'
+gem 'sassc'
```

Comment:
What are these new gems for? They don't seem to be related to the change.

Reply:
Without these,  `rake build` would always result in a dependency error.

Reply:
on your system? For me, "rake clean build" just works without it. Are you using the readme fedora approach of the "rvm" approach?

Reply:
also, if you don't specify a verion number, does it use the "latest"? That's means our build could break from one day to the next. (and it's ruby, so it will)